x-pack/plugins/cloud_security_posture/server/create_indices/create_processor.ts

```diff
@@ -7,22 +7,19 @@
 import { transformError } from '@kbn/securitysolution-es-utils';
 import type { ElasticsearchClient, Logger } from '@kbn/core/server';
 import { IngestPutPipelineRequest } from '@elastic/elasticsearch/lib/api/types';
-import {
-  CSP_INGEST_TIMESTAMP_PIPELINE,
-  CSP_LATEST_FINDINGS_INGEST_TIMESTAMP_PIPELINE,
-} from '../../common/constants';
 
 /**
- * @param pipelineId - the pipeline id to create. If a pipeline with the same pipelineId already exists, nothing is created or updated.
+ * @param pipelineConf - ingest pipeline configuration
```

Comment:
nit: usually you order the params in the documentation according to the parameters order